Assets/_BForBoss/_Character/Scripts/DashLinesEffectBehaviour.cs

```diff
@@ -0,0 +1,54 @@
+//-----------------------------------------------------------------------
+// <copyright file="DashLinesEffectBehaviour.cs" company="PERIGON GAMES">
+//     Copyright (c) PERIGON GAMES. 2020 All Rights Reserved
+// </copyright>
+//-----------------------------------------------------------------------
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Perigon.Character
+{
+    public class DashLinesEffectBehaviour : MonoBehaviour
+    {
+        [SerializeField] [Range(0,1)] float _lerpInStrength = 0.1f;
+        [SerializeField] [Range(0,1)] float _lerpOutStrength = 0.1f;
+        [SerializeField] float _duration = 0.1f;
+        private float _currentTime;
+        
+        [SerializeField] [Range(0,1)] float _maxAlpha = 1f;
+        private Material _speedLineMaterial;
+        
+        private void Start()
+        {
+            _speedLineMaterial = GetComponent<RawImage>().material;
+            _speedLineMaterial.SetFloat("_alpha",  0);
```

Comment:
It's better to put any strings at the top of the class as a `const`. 
`private const string MATERIAL_ALPHA = "_alpha"`   . you can name it better, but something along those lines. 

So you just need to set it once, and if `_alpha` string needs to change, you just need ot change it once.

Reply:
Good point! Changed👍